device_doctor/lib/src/health.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
```

Comment:
should this file renamed as recovery? health are just the check of the status, the actions to bring the device to a given state are the recovery ones.